### PR TITLE
addded order page

### PR DESCRIPTION
**Pull request by chenasrika**
Hi Pushpesh,
Please review the changes for the order page features

### CODE REVIEW

File: src/pages/OrderPage.java [hunk 1]
```diff
@@ -0,0 +1,9 @@
+package pages;
+
+public class OrderPage {
+
+	public static void main(String[] args) {
+		System.out.println("Hello from OrderPage");
+	}
```
Comment (learnwithpushpesh) [inline src/pages/OrderPage.java:8]: Hi, Please add logic

File: src/tests/OrderPageTest.java [hunk 2]
```diff
@@ -0,0 +1,11 @@
+package tests;
+
+public class OrderPageTest {
+	
+public void test1() {
+		
+		System.out.println("test 1- OP");
+	}
```
Comment (learnwithpushpesh) [inline src/tests/OrderPageTest.java:10]: Hi, Please add test logic.

File: src/pages/OrderPage.java (hunk 1 above)
Comment (chenasrika) [reply src/pages/OrderPage.java:8]: Logic addded.

File: src/tests/OrderPageTest.java (hunk 2 above)
Comment (chenasrika) [reply src/tests/OrderPageTest.java:10]: Test Logic addded.

**Review by learnwithpushpesh — APPROVED** [review]
PR is approved

**Comment by chenasrika** [comment]
merging orderpage branch to master as PR is approved

**Pull request merged**